### PR TITLE
another sweep of parts of the basic 65 appendix and more standardisation

### PR DESCRIPTION
**Pull request by Impakt**

### CODE REVIEW

File: appendix-basic65.tex [hunk 1]
```diff
@@ -5355,7 +5350,7 @@ \subsection{LOADIFF}
 
 \begin{verbatim}
 convert <myImage.jpg> <myImage.ppm>
-ppmtoilbm -aga <myImage.pbm> > <myImage.iff>
+ppmtoilbm -aga <myImage.ppm> > <myImage.iff>
```
Comment (gurcei) [inline appendix-basic65.tex:5353]: Just wanted to confirm, you gave this a try and confirmed it worked? (I've never tried this before myself, so it's unfamiliar turf).

Ah ok, saw a post in discord from ubik that seems to confirm you're right.

File: appendix-basic65.tex [hunk 2]
```diff
@@ -5838,8 +5833,10 @@ \subsection{NEXT}
 \item [Token:] \$82
 \item [Format:] {\bf FOR} index {\bf =} start {\bf TO} end
 		[{\bf STEP} step] ... {\bf NEXT} [index]
-\item [Usage:] Terminates the definition
-               of a BASIC loop with an index variable.
+\item [Usage:] Marks the end of the BASIC loop with the associated with the given index variable. When
```
Comment (gurcei) [inline appendix-basic65.tex:5836]: Remove the first `with the` (there's two presently.

So should be: `Marks the end of the BASIC loop associated with the given index variable.`

File: appendix-basic65.tex [hunk 3]
```diff
@@ -5860,12 +5857,11 @@ \subsection{NEXT}
 \item [Remarks:] The {\bf index} variable after {\bf NEXT} is
                optional. If it is omitted, the variable
                for the current loop is assumed.
-               % i dont understand this part ^^ - impakt
                Several consecutive {\bf NEXT} statements may be
                combined by specifying the indexes in a comma
                separated list. The statements
-               {\bf NEXT I:NEXT J:NEXT K} and
-               {\bf NEXT I,J,K} are equivalent.
+               \screentext{NEXT I,NEXT J,NEXT K} and
```
Comment (gurcei) [inline appendix-basic65.tex:5863]: The commas you switched to should be reverted back to colons (as `NEXT I, NEXT J, NEXT K` would produce a syntax error)

File: appendix-basic65.tex [hunk 4]
```diff
@@ -1523,35 +1523,24 @@ \subsection{CHAR}
...
-                    \begin{tabular}{|c|l|}
-                        \hline
-                        1	& Up \\
-                        2	& Right \\
-                        4	& Down \\
-                        8	& Left \\
-                        \hline
-                    \end{tabular}
-                    \end{center}
+                \begin{itemize}
```
[the remark this answers is not in the document]
Comment (Impakt) [reply appendix-basic65.tex:1526]: fixed

**Review by Edilbert — APPROVED** [review]

File: appendix-basic65.tex (hunk 2 above)
Comment (Impakt) [reply appendix-basic65.tex:5836]: done

File: appendix-basic65.tex (hunk 3 above)
Comment (Impakt) [reply appendix-basic65.tex:5863]: oops

**Review by gurcei — APPROVED** [review]
All good :)

**Pull request merged**